lang/c++/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ cmake_minimum_required (VERSION 3.1)
 set (CMAKE_LEGACY_CYGWIN_WIN32 0)
 
 if (NOT DEFINED CMAKE_CXX_STANDARD)
-    set(CMAKE_CXX_STANDARD 11)
+    set(CMAKE_CXX_STANDARD 17)
```

Comment:
apache/avro seems to have this. Wondering if it makes sense to rebase on the apache master branch

Reply:
>apache/avro seems to have this. Wondering if it makes sense to rebase on the apache master branch

we should probably wait until after we cut the release, unless we need stuff from from upstream avro. cc @BenPope maybe he has a sense of what's been going on upstream?